File: Dagger2DependencyInjectionDemo/app/src/main/java/com/kt/std/dagger2dependencyinjectiondemo/MainActivity.java
package com.kt.std.dagger2dependencyinjectiondemo;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    Car car;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        CarComponent carComponent = DaggerCarComponent.create();
        car = carComponent.getCar();

//        EngineFlap engineFlap = new EngineFlap();
//        CarBattery carBattery = new CarBattery();
//        CarChassis carChassis = new CarChassis();
//        CarEngine carEngine = new CarEngine(engineFlap);

//        Car car = new Car(carEngine, carChassis, carBattery);
        car.move();


    }
}

File: Rentateamtest/app/src/main/java/com/kt/std/rentateamtest/activities/MainActivity.kt
package com.kt.std.rentateamtest.activities

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.kt.std.rentateamtest.R
import com.kt.std.rentateamtest.fragments.UserListFragment

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        if (savedInstanceState == null) {
            supportFragmentManager
                .beginTransaction()
                .add(R.id.container, UserListFragment(), null)
                .commit()
        }
    }
}